src/Client.js

```diff
@@ -475,6 +479,27 @@ class Client extends EventEmitter {
 
         if (internalOptions.sendMediaAsSticker && internalOptions.attachment) {
             internalOptions.attachment = await Util.formatToWebpSticker(internalOptions.attachment);
+            if (options.stickerName || options.stickerAuthor) {
+                const exifPath = 'data.exif';
+                const resultPath = 'sticker.webp';
+                internalOptions.attachment = await Util.formatToWebpSticker(internalOptions.attachment);
+                fs.writeFileSync(resultPath, internalOptions.attachment.data, 'base64');
+                const random_id = Math.floor(Math.random() * (9999999999 - 1000000000) + 1000000000);
+                const stickerpackid = "com.marsvard.stickermakerforwhatsapp.stickercontentprovider "+ random_id;
+                const packname = options.stickerName || "undefined";
+                const author = options.stickerAuthor || "undefined";
+                const googlelink = "https://play.google.com/store/apps/details?id=com.marsvard.stickermakerforwhatsapp";
+                const applelink = "https://itunes.apple.com/app/sticker-maker-studio/id1443326857";
```

Comment:
I don't think every sticker should have information of another person's app (stickerpackid and links to google and apple store). Maybe passing theses throught options along with name and author might be a good option. 

If you default them to null author and name still works?

Reply:
WhatsApp provides the following exif data, since sticker makers are only probably available to apps via whatsapp's boilerplate (https://github.com/WhatsApp/stickers)
Not providing will probably chance an account getting banned?

Reply:
I will have to look more into that, as you are probably most right.. also if sticker author or name is specified then it adds that.

Reply:
I don't think this will increase the probabilities of getting banned because at the moment we are not providing any of that info

Reply:
Okay then, removing that.